lib/node_modules/@stdlib/blas/base/cscal/lib/cscal.js

```diff
@@ -0,0 +1,96 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var reinterpret = require( '@stdlib/strided/base/reinterpret-complex64' );
+
+
+// MAIN //
+
+/**
+* Scales a complex single-precision floating-point vector by a single-precision floating point constant.
+*
+* @param {PositiveInteger} N - number of indexed elements
+* @param {Complex64Array} cx - constant
```

Comment:
This is not how we'd provide a constant. We'd provided a complex number, just as we would in C, not a complex number array.